### DISCUSSION TITLE
Taiko scroll speed change every time edit the beatmap

### DISCUSSION THREAD

**Discussion by gagahpangeran** [ppy/osu#15424] [Q&A] [answered]
Reproduce in latest release:

**Original**

https://user-images.githubusercontent.com/4964985/139917396-30e3ebd4-5e2b-4880-8439-f25eed96b068.mp4

**After edit several times**

https://user-images.githubusercontent.com/4964985/139917425-028657bc-7f24-4116-bebe-a5a0c769410b.mp4

**Full repro step**

https://user-images.githubusercontent.com/4964985/139917495-a308f227-020e-4a64-a4b5-877a7f0e29f9.mp4

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15435
